scripts/src/main/java/io/github/fate_grand_automata/scripts/modules/attack/AvoidChainHandler.kt

```diff
@@ -0,0 +1,130 @@
+package io.github.fate_grand_automata.scripts.modules.attack
+
+import io.github.fate_grand_automata.scripts.enums.CardTypeEnum
+import io.github.fate_grand_automata.scripts.models.FieldSlot
+import io.github.fate_grand_automata.scripts.models.NPUsage
+import io.github.fate_grand_automata.scripts.models.ParsedCard
+import io.github.fate_grand_automata.scripts.models.toFieldSlot
+import io.github.lib_automata.dagger.ScriptScope
+import javax.inject.Inject
+import kotlin.collections.set
+
+@ScriptScope
+class AvoidChainHandler @Inject constructor(
```

Comment:
when AttackUtils is converted to object, we can also convert this to object.

Making it so we can just directly call the AttackUtils